lib/ofx/parser/transaction.ex

```diff
@@ -21,6 +22,7 @@ defmodule Ofx.Parser.Transaction do
 
     %{
       check_number: get(xml, @check_number),
+      ref_num: get(xml, @ref_num),
```

Comment:
It would be nice add a test for this new field in `test/ofx/parser/transaction_test.exs`